### ISSUE TITLE
Automatic upload of cancer wgs cases to genotype

### ISSUE THREAD

**Issue by Vince-janv** [Clinical-Genomics/cg#1544]
## Description

Now that cancer cases can be uploaded to genotype it would be nice to have automation in place to upload them similar to what we have for scout.

## Suggested solution

When uploading balsamic, check if WGS and if it contains a normal sample. If so run cg upload genotypes <case_id>

## This can be closed when

*Describe what needs to be done for this issue to be closed*

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by Vince-janv** [comment]
@GustavLagneborgScilifelab Maybe this can be a good first PR for you? @ivadym what do you think?

**Comment by GustavLagneborgScilifelab** [comment]
I would be interested in developing this!

**Comment by ivadym** [comment]
> I would be interested in developing this!

Great! 🥇

**Comment by henrikstranneheim** [comment]
Done here: https://github.com/Clinical-Genomics/cg/pull/1555
